com.unity.shadergraph/Documentation~/Compute-Deformation-Node.md

```diff
@@ -0,0 +1,12 @@
+# Compute Deformation Node
+
+## Description
+
+This node lets you pass compute deformed vertex data to a vertex shader, and only works with the [DOTS Hybrid Renderer](https://docs.unity3d.com/Packages/com.unity.rendering.hybrid@latest/). You must provide `DeformedVertexData` in the `_DeformedMeshData` buffer. The node uses the `_ComputeMeshIndex` property to calculate where the `DeformedVertexData` associated with the current mesh are located in the `_DeformedMeshData` buffer. To output data, you must either install both the DOTS Hybrid Renderer and DOTS Animation packages, or use a custom solution.
```

Comment:
Is there anything specific inside of the DOTS Hyrbid or Animation packages that goes in depth on how to use the Deformed Mesh Data buffer, or in what case this node would be useful? 
I'm just thinking of the case where a user stumbles across this node, having more information is better. We previously had some confusion regarding the linear blend skinning node.

Reply:
Yes we do :) It can be found [here](https://github.com/Unity-Technologies/dots/blob/dots-deformations/Samples/Packages/com.unity.rendering.hybrid/Documentation~/mesh_deformations.md). Please let us know if you think there is anything missing from the docs.

Reply:
That page looks good, can you make a note to add a link to it once it's public to users? I see it's in a private repo, but I would like the pages for skinning to have something along the lines of `These nodes work with the [DOTS Mesh Deformation](link to mesh deform page) system.`
Being that it's currently private I'm okay to approve without the link but I do want the public link added when it's available. c: